.github/workflows/celo-monorepo.yml

```diff
@@ -358,35 +251,24 @@ jobs:
             command: |
               yarn --cwd packages/protocol test stability/
     steps:
-      - uses: actions/cache/restore@v3
-        id: cache_git
-        with:
-          path: .git
-          key: git-${{ github.ref }}
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           submodules: recursive
       - name: Sync workspace
         uses: ./.github/actions/sync-workspace
         with:
-          package-json-checksum: ${{ needs.install-dependencies.outputs.package-json-checksum }}
           rebuild-package: 'true'
```

Comment:
do we need to rebuild here? 
We start with dependencies and compile artifacts from the restore/checkout, so doesn't seem like it

Reply:
checkon other branch and it seems it's needed